src/services/Api.php

```diff
@@ -282,6 +282,52 @@ public function profileInList($listId, $email): int
         return sizeof($content->data) > 0;
     }
 
+    /**
+     * getProfileFromList.
+     *
+     * @author	Unknown
+     * @since	v0.0.1
+     * @version	v1.0.0	Monday, November 14th, 2022.
+     * @access	public
+     * @param	mixed	$listId	
+     * @param	mixed	$email 	
+     * @return	object
+     */
+    public function getProfileFromList($listId, $email)
```

Comment:
Need to include Typing on this method. Indicate what type of thing the params are, and the type of thing that is being returned.